go.mod

```diff
@@ -3,15 +3,21 @@ module github.com/cloudwego/dynamicgo
 go 1.16
 
 require (
-	github.com/apache/thrift v0.13.0
+	github.com/apache/thrift v0.16.0
 	github.com/bytedance/sonic v1.8.8
 	github.com/chenzhuoyu/base64x v0.0.0-20221115062448-fe3a3abad311
 	github.com/cloudwego/kitex v0.6.1
 	github.com/cloudwego/thriftgo v0.2.11
 	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc
 	github.com/fatih/structtag v1.2.0
 	github.com/iancoleman/strcase v0.2.0
+	github.com/jhump/protoreflect v1.15.2
 	github.com/klauspost/cpuid/v2 v2.2.4
-	github.com/stretchr/testify v1.8.2
+	github.com/stretchr/testify v1.8.4
+	google.golang.org/protobuf v1.31.0
 
 )
+
+replace github.com/apache/thrift => github.com/apache/thrift v0.13.0
+
+replace github.com/stretchr/testify => github.com/stretchr/testify v1.8.2
```

Comment:
Why lock to v1.8.2 ?

Reply:
Well, there is no need to lock this, We just want to make the same with master branch, I will fix this later.